web-app/lib/tasks/api_weather_hazards.rake

```diff
@@ -0,0 +1,209 @@
+                        ring.each do |coordenate|
+                          if  GeographicManager.point_in_polygon(coordenate[0], coordenate[1])
+                             puts "*****California ALERT*****"  
+                             puts "type: #{warnings}"
+                             puts "start_date: #{Time.at(start_date/1000)}"
+                             puts "end_date: #{Time.at(end_date/1000)}"
+                           else
+                            puts "#{warnings} has values but not in Cali"
+                          end
+                        end
+                    end
+                end
+            else
+                puts "#{warnings} has not values"
+            end
+            
+        end
+    rescue Exception => ex
+        puts ex.message
+    end    
+  end
+end
+
+namespace :get do
+  task :watches_warning => :environment do
+    begin
+        watches_warnings=[]
+        url = "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Forecasts_Guidance_Warnings/watch_warn_adv/MapServer/1?f=pjson"
+        json = JSON.load(open(url))
+        json['drawingInfo']['renderer']['uniqueValueInfos'].each do |info|
+          watches_warnings << info['label']
+        end
+        watches_warnings.each do |warnings|
+            url = "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Forecasts_Guidance_Warnings/watch_warn_adv/MapServer/1/query?where=&text=#{warnings.to_param}&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&relationParam=&outFields=&returnGeometry=true&maxAllowableOffset=&geometryPrecision=&outSR=&returnIdsOnly=false&returnCountOnly=false&orderByFields=&groupByFieldsForStatistics=&outStatistics=&returnZ=false&returnM=false&gdbVersion=&returnDistinctValues=false&f=pjson"
+            json = JSON.load(open(url))
+            if json['features'].count > 0
+                json['features'].each do |feature|
+                    end_date = feature['attributes']['end_date']
+                    start_date = feature['attributes']['start_date']
+                    feature['geometry']['rings'].each do |ring|
+                        ring.each do |coordenate|
+                          if  GeographicManager.point_in_polygon(coordenate[0], coordenate[1])
+                             puts "*****California ALERT*****"  
+                             puts "type: #{warnings}"
+                             puts "start_date: #{Time.at(start_date/1000)}"
+                             puts "end_date: #{Time.at(end_date/1000)}"
+                           else
+                            puts "#{warnings} has values but not in Cali"
+                          end
+                        end
+                    end
+                end
+            else
+                puts "#{warnings} has not values"
+            end
+            
+        end
+    rescue Exception => ex
+        puts ex.message
+    end    
+  end
+end
```

Comment:
You should add some unit tests for these tasks using fake JSONs just to make sure you're parsing stuff correctly.